utils/swift_build_support/swift_build_support/products/tensorflow.py

```diff
@@ -176,73 +201,126 @@ def should_install(self, host_target):
         return self.args.build_tensorflow_swift_apis
 
     def install(self, host_target):
-        (suffixed_lib_name, unsuffixed_lib_name) = \
-            self._get_tensorflow_library(host_target)
+        # Install `libtensorflow.{dylib,so}` in the toolchain.
+        (suffixed_lib_name, unsuffixed_lib_name) = self._get_tensorflow_library(
+            host_target
+        )
 
         subdir = None
-        if host_target.startswith('macsox'):
-            subdir = 'macosx'
-        if host_target.startswith('linux'):
-            subdir = 'linux'
-
+        if host_target.startswith("macsox"):
+            subdir = "macosx"
+        if host_target.startswith("linux"):
+            subdir = "linux"
         if not subdir:
-            raise RuntimeError('unknown host target {}'.format(host_target))
+            raise RuntimeError("unknown host target {}".format(host_target))
 
         try:
-            os.unlink(os.path.join(self.install_toolchain_path(),
-                                   'usr', 'lib', 'swift',
-                                   subdir, suffixed_lib_name))
-            os.makedirs(os.path.join(self.install_toolchain_path(),
-                                     'usr', 'lib', 'swift', subdir))
+            os.unlink(
+                os.path.join(
+                    self.install_toolchain_path(),
+                    "usr",
+                    "lib",
+                    "swift",
+                    subdir,
+                    suffixed_lib_name,
+                )
+            )
+            os.makedirs(
+                os.path.join(
+                    self.install_toolchain_path(), "usr", "lib", "swift", subdir
+                )
+            )
```

Comment:
This needs to be split into two `try`/`catch` block is my suspicion - unlink may throw an `OSError` if the path does not exist, which will prevent `makedirs` from being executed.

Reply:
I'll defer this since this code wasn't introduced in this patch.